DogScepterLib/Core/GMData.cs

```diff
@@ -78,7 +78,7 @@ public class GMVersionInfo
         public bool RoomObjectPreCreate = false;
 
         /// <summary>
-        /// Whether the GameMaker data file contains various variable counts whose purpose is currently unknown.
+        /// Whether some unknown variables in the GameMaker data file have different values.
```

Comment:
Oh, this field is referring to the variable counts in one of the chunks (I forget exactly where). There's separate fields for instance/global variables, and this is `false` when those values end up being always the same anyway (on certain GameMaker versions). The counts, I believe, are used to instantiate the correct amount of memory for GML variables, though some may be unused as well.

Reply:
Oh I forgot to specify the chunk whoops.
They're in "VARI", I'm aware of the usage of this value, but i probably should've clarified them a bit more.